opendistro-elasticsearch-jdbc.release-notes

```diff
@@ -1,5 +1,11 @@
+## 2019-06-06, Version 1.0.0 (Current)
 
-## 2019-04-19, Version 0.9.0 (Current)
+### Changes
+
+* Update to LocalDateTime conversion to Timestamp to support timezone [issue #6]
```

Comment:
NP: Rewording to "Update```d``` to LocalDateTime conversion ```for``` Timestamp to support timezone" may be better here

Reply:
I initially had "Updated the ... conversion" but then saw the previous entries were nouns, not verbs, so changed to noun.

Reply:
I noticed the one below it says "Updated"

Reply:
:)) You caught me. So which one do you want me to change? Does it even matter?

Reply:
Either is fine, no preference on my end :)